pkg/extensions/tbtc/tbtc.go

```diff
@@ -420,6 +435,10 @@ func (t *tbtc) monitorAndAct(
 	timeoutFn timeoutFn,
 ) (subscription.EventSubscription, error) {
 	handleStartEvent := func(depositAddress string) {
+		if !monitoringFilterFn(depositAddress) {
```

Comment:
Maybe `shouldMonitorFn`? I see `monitoringFilterFn` is more consistent with other `monitoring*` functions but `!monitoringFilterFn(...)` reads weird. My first thought was that this is the case for a situation when this function is not defined.

Reply:
Also... why do we pass it as a parameter, can't we just call `shouldMonitorDeposit` directly? It's always the same function that is passed here.

Reply:
Ok, I'll rename. However, I'd vote to leave it as a parameter. It's a more flexible approach and fits better with the generic spirit of the `monitorAndAct` function.